src/Illuminate/Foundation/Bootstrap/RegisterFacades.php

```diff
@@ -17,12 +21,51 @@ class RegisterFacades
      */
     public function bootstrap(Application $app)
     {
-        Facade::clearResolvedInstances();
+        Facades\Facade::clearResolvedInstances();
 
-        Facade::setFacadeApplication($app);
+        Facades\Facade::setFacadeApplication($app);
 
         AliasLoader::getInstance(array_merge(
-            $app->make('config')->get('app.aliases', []),
+            $app->make('config')->get('app.aliases', [
```

Comment:
Can't we move the core aliases above this line in `array_merge`? 
Thus, when adding our custom aliases, we do not have to copy this, as Taylor wrote above.

Reply:
That means there is no way to totally disable aliases.

Reply:
Yes, but as the author of this PR noted, they are basically not modified by most people in any way.

Reply:
@igorgaming After discussing with Taylor internally, we have applied your suggestion.